fhir-bucket/src/main/resources/stats/postgres/vacuum_full.sql

```diff
@@ -0,0 +1,72 @@
+VACUUM FULL fhirdata.CarePlan_str_values;
```

Comment:
what does this mean?

Reply:
VACUUM FULL is only needed when you have a table that is mostly dead rows - ie, the vast majority of its contents have been deleted. It should not be used for table optimization or periodic maintenance, as it's generally counterproductive. In most cases the freed space will be promptly re-allocated, possibly increasing file-system-level fragmentation and requiring file system space allocations that're slower than just re-using existing free space within a table. 

I really think these need comments as to the purposes of these sqls

Reply:
This was just working SQL I applied after the big reindex of 2020 in order to see what the space looked like after. Recorded for posterity.